image/cas/interface.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016 The Linux Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package cas implements generic content-addressable storage.
+package cas
```

Comment:
On Wed, Jul 27, 2016 at 02:16:23PM -0700, Brandon Philips wrote:
> @wking Can you add a note that says "This CAS interface is for
> internal use of oci-image-tool for the time being. It is subject to
> change. This notice will be removed when and if the interface
> becomes stable."

Done in 059d807.